### DISCUSSION TITLE
请问脚本notify_media.lua可以删除吗？

### DISCUSSION THREAD

**Discussion by wli189** [dyphire/mpv-config#30] [Q&A] [answered]
不知道这个是不是重要主件，会造成不能同步播放记录。
不与其他关联，那我就删除了。

**Comment by dyphire** [comment, marked as answer]
notify_media.lua 不是什么重要组件，可以自行删除
就如 scripts 文件夹里 [README.md](https://github.com/dyphire/mpv-config/tree/master/scripts#readme) 中说明的那样，这个脚本是用来实现 win10/11 上的媒体 SMTC 功能，觉得不需要是可以精简的
同时它还变相为蓝牙耳机实现了控制功能（mpv 原生并不支持
